### PR TITLE
refactor: Use Q instead of client.all

### PR DESCRIPTION
**Pull request by Crash--**

### CODE REVIEW

File: src/drive/web/modules/navigation/FileExplorer.jsx [hunk 1]
```diff
@@ -219,12 +219,11 @@ export const handleFileOpen = (file, availableOffline, props, dispatch) => {
   }
 }
 
-const appsQuery = client => client.all('io.cozy.apps')
 export default compose(
   translate(),
   queryConnect({
     apps: {
-      query: appsQuery,
+      query: Q('io.cozy.apps'),
```
Comment (ptbrowne) [inline src/drive/web/modules/navigation/FileExplorer.jsx]: I think you have to do `() => Q('io.cozy.apps')`

Comment (Crash--) [reply src/drive/web/modules/navigation/FileExplorer.jsx]: Hmm it seems to work without  🤔

Comment (ptbrowne) [reply src/drive/web/modules/navigation/FileExplorer.jsx]: Ah oui non c'est OK https://github.com/cozy/cozy-client/blob/master/packages/cozy-client/src/Query.jsx#L28-L29

**Comment by ptbrowne** [comment]
LGTM

**Pull request merged**